azure/azure_template.yaml

```diff
@@ -34,69 +29,80 @@ parameters:
   - name: IMAGE
     type: string
     default: public
-  - name: SKIP_TESTS
+  - name: COVERAGE
     type: boolean
     default: false
-  - name: TIMEOUT
+  # Style checks
+  - name: TIMEOUT_STYLE
     type: number
-    default: 0
-  - name: COVERAGE
+    default: 10
+  - name: IGNORE_STYLE
     type: boolean
     default: false
+  - name: ISORT
+    type: boolean
+    default: false
+  - name: PYLINT
+    type: boolean
+    default: false
+  - name: CLANG_FORMAT
+    type: boolean
+    default: false
+  # Tapenade
   - name: TAPENADE
     type: boolean
     default: false
+  - name: TIMEOUT_TAPENADE
+    type: number
+    default: 10
   - name: TAPENADE_SCRIPT
     type: string
     default: .github/build_tapenade.sh
   - name: TAPENADE_VERSION
     type: string
     default: 3.10
-  - name: CLANG_FORMAT
-    type: boolean
-    default: false
```

Comment:
I think I may be confused with how this is set up now, but why are these options (`CLANG_FORMAT` and the ones at the top) removed? Are they no longer passes into this file?

Reply:
The diff is a bit confusing. I just moved them around to get the order consistent with the stages (build, style, Tapenade).

Reply:
Got it. Looks good to me!